test/pydmap_tests/test_dmapread.py

```diff
@@ -0,0 +1,325 @@
+# Copyright (C) 2018  SuperDARN
+# Authors: Keith Kotyk and Marina Schmidt
+
+"""
+This test suite is to test the implementation of the DmapRead class that reads SuperDARN
+DMAP files: rawacf, fitacf, map, ... etc
+"""
+
+import unittest
+import numpy as np
+import logging
+import collections
+
+import pydarn
+
+pydarn_logger = logging.getLogger('pydarn')
+
+# Test files
+rawacf_file = "./testfiles/20170410.1801.00.sas.rawacf"
```

Comment:
I think these files should be trimmed to only a few records. We dont need full 2hr files.

Reply:
Yes... again out of scope as this will be looked at in test suite branch.